### PR TITLE
Document thread-safety of a service

### PR DESCRIPTION
**Pull request by jagij**
A service should be implemented in a thread-safe way. A comment on the thread safety of the example service has been added.

### CODE REVIEW

**Comment by jagij** [comment]
I signed it!

File: docs/tutorials/basic/c.md [hunk 1]
```diff
@@ -252,6 +252,12 @@ information, and then `return` with an `OK` status to tell gRPC that we've
 finished dealing with the RPC and that the `Feature` can be returned to the
 client.
 
+Note that all service methods can (and will!) be called from multiple threads at
+the same time. So you have to make sure that your method implementations are
```
Comment (dgquintas) [inline docs/tutorials/basic/c.md]: Remove "So".

File: docs/tutorials/basic/c.md [hunk 2]
```diff
@@ -252,6 +252,12 @@ information, and then `return` with an `OK` status to tell gRPC that we've
...
+thread safe. In our example case here, `feature_list_` is never changed after
```
Comment (dgquintas) [inline docs/tutorials/basic/c.md]: remove "case here"

**Comment by jagij** [comment]
Thanks for the feedback! I made changes accordingly.
It seems I used a different e-mail address last time, so I amended the commit with the already accepted CLAs, and force pushed it. Hope that doesn't break things..

**Pull request merged**